build_debian.sh

```diff
@@ -271,9 +271,11 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install docker-ce=${DOCKER_VERSIO
 
 install_kubernetes () {
     local ver="$1"
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install gnupg
     sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT curl -fsSL \
         https://packages.cloud.google.com/apt/doc/apt-key.gpg | \
         sudo LANG=C chroot $FILESYSTEM_ROOT apt-key add -
```

Comment:
`apt-key` is deprecated and may be removed soon. The suggestion is to just copy the file that is downloaded to `/etc/apt/trusted.gpg.d/`. (Note that you may need to change the suffix from `.gpg` to `.asc`, depending on the actual key format that is used.)

Reply:
This will also remove the gnupg dependency.

Reply:
Thanks, very helpful, changed the code as your comments.